src/tests/testTokenizer.cc

```diff
@@ -248,6 +248,68 @@ TestTokenizer::testTokenizerInt64()
         CPPUNIT_ASSERT(t.buf().isEmpty());
     }
 
+    // autodetect octal base in minimal valid input
+    {
+        int64_t rv;
+        Parser::Tokenizer t(SBuf("0"));
+        const int64_t benchmark = 0;
+        CPPUNIT_ASSERT(t.int64(rv));
+        CPPUNIT_ASSERT_EQUAL(benchmark,rv);
+        CPPUNIT_ASSERT(t.buf().isEmpty());
+    }
```

Comment:
I tried to add an equivalent case earlier, when enumerating all possible interesting short cases. Look for "auto-detected base; input: base, no post-base digits, unlimited". That case had wrong base (8 instead of zero). Fixed in c409c333.

Reply:
Thank you.